test/functional/rpc_rawtransaction.py

```diff
@@ -487,6 +486,44 @@ def run_test(self):
         assert_equal(testres['allowed'], True)
         self.nodes[2].sendrawtransaction(hexstring=rawTxSigned['hex'], maxfeerate='0.20000000')
 
+        self._test_searchrawtransactions()
+
+    def _test_searchrawtransactions(self):
```

Comment:
Please also test the spends_results path (returning transactions which spend from this address).